packages/components/text/src/text.spec.js

```diff
@@ -1,9 +1,12 @@
 import React from 'react';
-import invariant from 'tiny-invariant';
+import { invariant } from '@commercetools-uikit/utils';
 import { screen, render } from '../../../../test/test-utils';
 import Text from './text';
 
-jest.mock('tiny-invariant');
+jest.mock('@commercetools-uikit/utils', () => ({
+  ...jest.requireActual('@commercetools-uikit/utils'),
+  invariant: jest.fn(),
```

Comment:
Now or later: the `invariant` could have a `__mock__`. Then you could just to `jest.mock()` maybe.

Reply:
I'll leave this as homework